addons/sourcemod/scripting/shop.sp

```diff
@@ -1198,6 +1202,238 @@ public int ItemPanel_Handler(Menu menu, MenuAction action, int param1, int param
 	}
 }
 
+void ConfirmBuy(int client, int item_id)
+{
+	Panel panel = ItemManager_ConfirmItemPanelInfo(client, item_id, Menu_Buy, true);
+	if (panel != null)
+	{
+		char sBuffer[SHOP_MAX_STRING_LENGTH], sItemId[16];
+		IntToString(item_id, sItemId, sizeof(sItemId));
+		
+		bool isHidden = ItemManager_GetItemHideEx(sItemId);
+		
+		SetGlobalTransTarget(client);
+		
+		int credits = GetCredits(client);
+		
+		FormatEx(sBuffer, sizeof(sBuffer), "%t\n ", "credits", credits);
+		panel.SetTitle(sBuffer, false);
+		
+		ItemType type = ItemManager_GetItemTypeEx(sItemId);
+	
+		panel.DrawItem(" ", ITEMDRAW_SPACER|ITEMDRAW_RAWLINE);
+		
+		switch (type)
+		{
+			case Item_None :
```

Comment:
Actually there are no items with this type of item. It is left for compatibility only. @R1KO am I right?